charm/kafka/reactive/kafka.py

```diff
@@ -166,7 +185,8 @@ def storage_attach():
     log_dir = os.path.join(mount, "logs")
     unitdata.kv().set('kafka.storage.log_dir', log_dir)
     hookenv.log('Kafka logs storage attached at {}'.format(log_dir))
-    # Stop Kafka; removing the kafka.started state will trigger a reconfigure if/when it's ready
+    # Stop Kafka; removing the kafka.started state will trigger \
```

Comment:
Could remove the trailing ` \` for a comment.

Reply:
done